src/app/access-control/epeople-registry/eperson-form/eperson-form.component.html

```diff
@@ -44,68 +40,65 @@ <h1 class="border-bottom pb-2">{{messagePrefix + '.create' | translate}}</h1>
-                    <tr>
-                      <th scope="col" class="align-middle">{{messagePrefix + '.table.id' | translate}}</th>
-                      <th scope="col" class="align-middle">{{messagePrefix + '.table.name' | translate}}</th>
-                      <th scope="col" class="align-middle">{{messagePrefix + '.table.collectionOrCommunity' | translate}}</th>
-                    </tr>
-                  </thead>
-                  <tbody>
-                    @for (group of (groups$ | async)?.payload?.page; track group) {
-                      <tr>
-                        <td class="align-middle">{{group.id}}</td>
-                        <td class="align-middle">
-                          <a (click)="groupsDataService.startEditingNewGroup(group)"
-                            [routerLink]="[groupsDataService.getGroupEditPageRouterLink(group)]">
-                            {{ dsoNameService.getName(group) }}
-                          </a>
-                        </td>
-                        <td class="align-middle">
-                          {{ dsoNameService.getName((group.object | async)?.payload) }}
-                        </td>
-                      </tr>
-                    }
-                  </tbody>
-                </table>
-              </div>
-            </ds-pagination>
-          }
-          @if ((groups$ | async)?.payload?.totalElements === 0) {
-            <div class="alert alert-info w-100 mb-2" role="alert">
-              <div>{{messagePrefix + '.memberOfNoGroups' | translate}}</div>
-              <div>
-                <button [routerLink]="[groupsDataService.getGroupRegistryRouterLink()]"
-                class="btn btn-primary">{{messagePrefix + '.goToGroups' | translate}}</button>
-              </div>
-            </div>
-          }
+      <div>
+        <h2>{{messagePrefix + '.groupsEPersonIsMemberOf' | translate}}</h2>
+        @if (groups$ | async | dsHasNoValue) {
+        <ds-loading [showMessage]="false"></ds-loading>
+        }
+        @if ((groups$ | async)?.payload?.totalElements > 0) {
+        <ds-pagination [paginationOptions]="config" [collectionSize]="(groups$ | async)?.payload?.totalElements"
+          [hideGear]="true" [hidePagerWhenSinglePage]="true" (pageChange)="onPageChange($event)">
+          <div class="table-responsive">
+            <table id="groups" class="table table-striped table-hover table-bordered">
+              <thead>
+                <tr>
+                  <th scope="col" class="align-middle">{{messagePrefix + '.table.id' | translate}}</th>
+                  <th scope="col" class="align-middle">{{messagePrefix + '.table.name' | translate}}</th>
+                  <th scope="col" class="align-middle">{{messagePrefix + '.table.collectionOrCommunity' | translate}}
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                @for (group of (groups$ | async)?.payload?.page; track group) {
```

Comment:
All the changes in this file appear to be code realignment.  We don't see where you've added the delete button to this page?  Is there code missing?

You should remove all the alignment changes (i.e. undo all current changes to this file), and double check that the delete button has been added into this HTML.